.travis.yml

```diff
@@ -1,5 +1,8 @@
-language: node
+sudo: required
+dist: trusty
+language: node_js
 
 script:
   - scripts/check-javascript-syntax-errors.sh
   - scripts/check-html-syntax-errors.sh
+  - bash scripts/deploy-surge.sh
```

Comment:
If you mark this script as executable with 

    chmod +x scripts/deploy-surge.sh

You do not need bash in front of it.